modules/nw-mutual-tls-auth.adoc

```diff
@@ -65,5 +65,5 @@ $ oc edit IngressController default -n openshift-ingress-operator
 [source,terminal]
 ----
 $ openssl  x509 -in custom-cert.pem  -noout -subject
-subject= /CN=example.com/ST=NC/C=US/O=Security/OU=OpenShift
+  subject= /CN=example.com/ST=NC/C=US/O=Security/OU=OpenShift
```

Comment:
```suggestion
subject=C=US, ST=NC, O=Security, OU=OpenShift, CN=example.com
```
This line is the output of `openssl x509...` command, so should not add the spaces, and please add "Example output" before this line. And also correct the output format.